keepalived.te

```diff
@@ -43,6 +43,7 @@ allow keepalived_t self:netlink_connector_socket create_socket_perms;
 allow keepalived_t self:netlink_route_socket nlmsg_write;
 allow keepalived_t self:packet_socket create_socket_perms;
 allow keepalived_t self:rawip_socket create_socket_perms;
+corenet_raw_bind_generic_node(keepalived_t)
```

Comment:
@mscherer could you move the line to the other corenet_ calls?

Reply:
yup